content/tbb/tbb-21/contents.lr

```diff
@@ -8,3 +8,8 @@ description:
 Click the button labelled "Copy Tor Log To Clipboard" that appears in the dialog window when Tor Browser is first connecting to the network.
 If Tor Browser is already open, click on the Torbutton icon (the small green onion at the top-left of the screen), then "Open Network Settings", then "Copy Tor Log To Clipboard".
 Once you have copied the log, you will be able to paste it into a text editor or email client.
+
+#### Advanced
+You can control how Tor logs information with the `Log` setting in your torrc file. Have a look at some common uses in [this torrc sample](https://gitweb.torproject.org/tor.git/tree/src/config/torrc.sample.in#n30) and the [Tor man page](https://2019.www.torproject.org/docs/tor-manual.html.en#Log) documentation for `Log`.
+
+Not sure how to edit your torrc? See [this question](https://2019.www.torproject.org/docs/faq#torrc) to learn how!
```

Comment:
We should stop ourselves from redirecting new documentation to the old TPO world. If that resource is important and still works, we should import to new TPO world (support, community, tpo, or hold to dev.tpo or docs.tpo). 
Some FAQ questions should be imported to support.tpo